client/a8c-for-agencies/sections/marketplace/common/referral-toggle/index.tsx

```diff
@@ -23,6 +24,15 @@ const ReferralToggle = () => {
 
 	const isAgencyApproved = useSelector( hasApprovedAgencyStatus );
 
+	const handleToggle = () => {
+		toggleMarketplaceType();
+		dispatch(
+			recordTracksEvent( 'calypso_a4a_marketplace_referral_toggle', {
+				purchase_mode: marketplaceType === 'referral' ? 'regular' : 'referral',
```

Comment:
Do we need to use the Elvis operator here, or can we just pass `marketplaceType` directly?